docs/getting_started/general/AWS.md

```diff
@@ -59,41 +59,51 @@ aws ec2 authorize-security-group-ingress \
 ```
 
 ## Spinning up the VMs
-Create the 2 Calico Docker hosts by passing in a `cloud-config` file. A different file is used for the first server. 
+Create the 2 Calico Docker hosts by passing in a `cloud-config` file. A different file is used for the first server. Copy the two files located at `vagrant-coreos/user-data-first` and `vagrant-coreos/user-data-others` onto your machine.
 
-For the first server, use the file located in `vagrant-coreos/user-data-first`
-For the second server, use the file located in `vagrant-coreos/user-data-other`
+Open your `vagrant-coreos/user-data-others` file and replace the two instances of `calico-01` with `localhost`.
```

Comment:
Is this meant to say first no others?

And is there a reason we can't do this in the source file?